### DISCUSSION TITLE
Trying to run llama-server, but `llama_model_load: error loading model: missing tensor 'token_embd.weight'`

### DISCUSSION THREAD

**Discussion by MichelleTanPY** [ggml-org/llama.cpp#10813] [Q&A] [answered]
Hi everyone, I'm new to this repo and trying to learn and pick up some easy issue to contribute to. I'm following all the steps in this [README ](https://github.com/ggerganov/llama.cpp/tree/master/examples/server), trying to run llama-server locally, but I ended up with this error: `llama_model_load: error loading model: missing tensor 'token_embd.weight'`. I've tried all the .gguf in models server, and was wondering if the README is slightly dated as I don't see this `models\7B\ggml-model.gguf` as well.
Appreciate anyone's time on explaining this.

**Comment by MichelleTanPY** [comment, marked as answer]
I don't completely understand why. But the *.gguf models in the models/ folder doesn't seem to work. I downloaded another model, that manage to get the llama-server up and running.

**Reply by slaren** [reply]
If you mean the `ggml-vocab-*.gguf` files, these only contain the tokenizers and are meant to be used with the tests only. I think it would be good to move them to `/tests` to avoid confusion.

**Reply by MichelleTanPY** [reply]
@slaren  Yes, it was quite confusing. Thanks for clarifying. Will it be ok if I submit a PR for moving these under a new test directory within models? `/models/tests`

**Reply by slaren** [reply]
Yes, `/models/tests` sounds good to me.
